src/models/users/usePatchUser.js

```diff
@@ -1,9 +1,13 @@
 import { useState } from 'react';
 import axios from 'axios';
 import { get } from 'lodash-es';
+import { useQueryClient } from 'react-query';
+
+import queryKeys from '../../constants/queryKeys';
 import { formatError } from '../../utils/formatters';
 
 export default function usePatchUser(userId) {
+  const queryClient = useQueryClient();
```

Comment:
I'm seeing that some of these custom hooks are using a QueryClient while others are not. I'm sure that this will become more clear to me once I read the documentation, but I find myself wondering why.

Reply:
it's just like when we do `const intl = useIntl();`